src/components/cleanButton.jsx

```diff
@@ -1,4 +1,4 @@
-export const CleanButton = () => {
+export const CleanButton = (cleanFields) => {
```

Comment:
not sure. it works? I think you should use {}.

Reply:
if you don't use {}, you can write (props), then props.cleanFields

Reply:
I fixed it. Now it works
Thnx!

Reply:
I don't see a new commit...